README.md

```diff
@@ -86,7 +88,7 @@ By default, the API response when retrieving a single record will be in the JSON
 
 ### Links
 
-The `links` object contains links to metadata related to this record but not directly included in the record (e.g. citation information), and alternative serialization formats (e.g. BibTeX).
+The `links` object contains links to metadata related to this record but not directly included in the record (e.g. citation information), and [alternative serialization formats](#changing-formats) (e.g. BibTeX).
```

Comment:
@tsgit commented:
>I would mention content-negotiation via Accept-Header more prominently. no
lenghty explanation required. It should be speaking for itself for anybody
attempting to use the API. simply point to some canonical resources
https://developer.mozilla.org/en-US/docs/Web/HTTP/Content_negotiation
https://restfulapi.net/content-negotiation/

I've linked to the relevant section of the docs...